website/docs/components/tabs/partials/guidelines/guidelines.md

```diff
@@ -47,6 +47,20 @@ Don’t trigger a Tab change via an external action, such as a submit or next bu
 ![Tabs with external trigger](/assets/components/tabs/tabs-external-trigger.png)
 !!!
 
+## Sizes
+
+Medium-sized tabs work best in most contexts. Large tabs can be used when needed to emphasize important sections, enhance clarity, and convey hierarchical distinction. 
+
+![Tabs sizes example](/assets/components/tabs/tab-size.png)
+
+### Choosing sizes 
+
+Mixing different tab sizes is not recommended, as it can confuse users. Choose tab sizes with a purpose, aligning them with the hierarchical importance of the information they represent. 
```

Comment:
Couple of suggestions here

```suggestion
Mixing different tab sizes is not recommended and can negatively impact the natural page hierarchy. Choose tab sizes intentionally and aligning them with the hierarchical importance of the information they represent. 
```

Reply:
Made an additional small grammar fix in the above suggestion.

"... **aligning** them with the hierarchical importance..."
=>
"... **align** them with the hierarchical importance..."